OAuth/ResourceOwner/GenericOAuth2ResourceOwner.php

```diff
@@ -12,6 +12,7 @@
 namespace HWI\Bundle\OAuthBundle\OAuth\ResourceOwner;
 
 use HWI\Bundle\OAuthBundle\Security\Core\Authentication\Token\OAuthToken;
+use ReflectionClass;
```

Comment:
Base PHP class should not be imported according to Symfony coding style.

Use `\ReflectionClass` instead.